src/app/features/activityLog/components/activities-log.component.html

```diff
@@ -0,0 +1,19 @@
+<mat-list id="activities-list">
+    <mat-list-item *ngFor="let activityLog of activities">
+        <div matLine fxLayout="row">
+            <div fxFlex>
+                <strong>{{activityLog.activityStatus}}</strong>
+            </div>
+            <div fxFlex>
+                {{activityLog.createdBy}}
+            </div>
+
+            <div fxFlex>
```

Comment:
formatting + question, shouldn't we have here some max boundaries?
like status 50% of available space or createdBy aligned to the right edge?